src/components/Main/CommandPalette/SearchSuggestionHistoryItem.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div :class="$style.container" @click="onClick">
+    <span :class="$style.label">{{ label }}</span>
+    <icon
+      name="close"
+      mdi
+      :class="$style.icon"
+      :size="24"
+      :inner-size="12"
+      @click.stop="removeHistorySuggestion(label)"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import Icon from '@/components/UI/Icon.vue'
+import { useCommandPaletteStore } from '@/providers/commandPalette'
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+  name: 'SearchSuggestionHistoryItem',
+  components: { Icon },
+  props: {
+    label: {
+      type: String,
+      required: true
+    }
+  },
+  emits: {
+    select: (label: string) => true //TODO 必要？
```

Comment:
```suggestion
    select: (label: string) => true
```
補完とかの都合上、明示するほうがいいです 🤘